diff --git a/node_modules/next-tinacms-cloudinary/dist/handlers.d.ts b/node_modules/next-tinacms-cloudinary/dist/handlers.d.ts
index cc62b42..07f85ef 100644
--- a/node_modules/next-tinacms-cloudinary/dist/handlers.d.ts
+++ b/node_modules/next-tinacms-cloudinary/dist/handlers.d.ts
@@ -15,6 +15,7 @@ export interface CloudinaryConfig {
     cloud_name: string;
     api_key: string;
     api_secret: string;
+    secure: true;
     authorized: (req: NextApiRequest, res: NextApiResponse) => Promise<boolean>;
 }
 export declare const mediaHandlerConfig: {
